modules/util/truncate.go

```diff
@@ -14,31 +14,92 @@ const (
 	asciiEllipsis = "..."
 )
 
-func IsLikelySplitLeftPart(s string) bool {
+func IsLikelyEllipsisLeftPart(s string) bool {
```

Comment:
Okay, yeah, I can see now where the `leftpart` is coming from.
I'm still not certain if we should include that in the method name.
But I guess it's fine for now.